.cbmc-batch/include/proof_helpers/utils.h

```diff
@@ -18,6 +18,12 @@
 #include <stddef.h>
 #include <stdint.h>
 
+struct store_byte_from_buffer {
+    size_t index;
+    uint8_t byte;
+};
+
 void assert_bytes_match(const uint8_t *a, const uint8_t *b, size_t len);
 void assert_all_bytes_are(const uint8_t *a, const uint8_t c, size_t len);
 void assert_all_zeroes(const uint8_t *a, size_t len);
+void assert_byte_from_buffer_matches(const uint8_t *buffer, struct store_byte_from_buffer *b);
```

Comment:
https://github.com/danielsn/aws-c-common/blob/hash_table_proofs/.cbmc-batch/source/utils.c#L47-L66
Consider importing these functions and using them instead